bridge/jni/src/mappings.rs

```diff
@@ -312,6 +314,7 @@ fn map_config_to_rust(config: proto_output::Config) -> SdkJniResult<RustConfig>
 		max_key_page_size_bytes: config.max_key_page_size_bytes,
 		dsnp_versions,
 		schema_map,
+		graph_public_key_schema_id: config.graph_public_key_schema_id as SchemaId,
```

Comment:
nit: don't think you need the `as SchemaId` here; `Config.graph_public_key_schema_id` _is_ a `SchemaId`

Reply:
this is probably confusing but here  `config` is protobuf config where graph key schema id is of type `uint32`

Reply:
Ah, okay